force-app/main/default/pages/STG_PanelContactRoles.page

```diff
@@ -52,9 +52,11 @@
             <div class="slds-form-element">
                 <apex:outputLabel value="{!$ObjectType.npo02__Households_Settings__c.Fields.npo02__Always_Rollup_to_Primary_Contact__c.Label}" for="cbxARPC" styleClass="slds-form-element__label" />
                 <div class="slds-form-element__control">
-                    <apex:inputCheckbox value="{!stgService.stgHH.npo02__Always_Rollup_to_Primary_Contact__c}" rendered="{!isEditMode}" id="cbxARPC" html-aria-describedby="{!$Component.cbxARPCHelp}" styleClass="slds-checkbox"/>
-                    <apex:inputCheckbox value="{!stgService.stgHH.npo02__Always_Rollup_to_Primary_Contact__c}" rendered="{!isReadOnlyMode}" disabled="true" id="cbxARPCO" html-aria-describedby="{!$Component.cbxARPCHelp}" styleClass="slds-checkbox"/>
-                    <apex:outputPanel id="cbxARPCHelp" layout="block">
+                    <label class="slds-checkbox">
+                        <apex:inputCheckbox value="{!stgService.stgHH.npo02__Always_Rollup_to_Primary_Contact__c}" rendered="{!isEditMode}" id="cbxARPC" html-aria-describedby="{!$Component.cbxARPCHelp}" styleClass="slds-checkbox"/>
+                        <apex:inputCheckbox value="{!stgService.stgHH.npo02__Always_Rollup_to_Primary_Contact__c}" rendered="{!isReadOnlyMode}" disabled="true" id="cbxARPCO" html-aria-describedby="{!$Component.cbxARPCHelp}" styleClass="slds-checkbox"/>
+                    <span class="slds-checkbox_faux"></span>
+                    </label>                                        <apex:outputPanel id="cbxARPCHelp" layout="block">
```

Comment:
Missing a new line here? Looks like there's a few of these, I'll let you find/fix the rest 😉